src/_pytest/_py/path.py

```diff
@@ -953,7 +953,7 @@ def ensure(self, *args, **kwargs):
         else:
             p.dirpath()._ensuredirs()
             if not p.check(file=1):
-                p.open("w").close()
+                p.open("wb").close()
```

Comment:
To just touch the file, pathlib uses the below pattern -- this might be a larger change than wanted, though:

```suggestion
                os.close(os.open(self.strpath, flags=os.O_CREAT))
```

Reply:
This code Was strictly copied in to enable drop of the py dep, so let's go for minimal changes instead of nice ones as we want to shed it as soon as possible